pyproject.toml

```diff
@@ -1,8 +1,86 @@
 [build-system]
-# AVOID CHANGING REQUIRES: IT WILL BE UPDATED BY PYSCAFFOLD!
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
-[tool.setuptools_scm]
-# See configuration details in https://github.com/pypa/setuptools_scm
-version_scheme = "no-guess-dev"
+[project]
+name = "SALib"
+dynamic = ["version"]
+description = "Tools for sensitivity analysis. Contains Sobol', Morris, and FAST methods"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = "MIT"
+authors = [
+  { name = "Jon Herman", email = "jdherman8@gmail.com" },
+  { name = "Will Usher" },
+]
+maintainers = [
+  { name = "SALib contributors" },
```

Comment:
Should we be more specific here and take the list from github?

Reply:
I would not really advise for it as it's a lot of additional maintenance. I think there are bots that could help do things like these. But usually projects care more about updating the release notes and have all the folks there. I suggest doing this elsewhere in any case as this PR is already packed with a lot of things.